src/pybind/mgr/dashboard/frontend/src/app/ceph/rgw/rgw-bucket-list/rgw-bucket-list.component.ts

```diff
@@ -88,6 +93,65 @@ export class RgwBucketListComponent extends ListWithDetails {
     this.timeConditionReached();
   }
 
+  ngOnInit() {
+    this.columns = [
+      {
+        name: this.i18n('Name'),
+        prop: 'bid',
+        flexGrow: 1
+      },
+      {
+        name: this.i18n('Owner'),
+        prop: 'owner',
+        flexGrow: 2
+      },
+      {
+        name: this.i18n('Size Usage'),
+        prop: 'size_usage',
+        cellTemplate: this.bucketSizeTpl,
+        flexGrow: 1
+      },
+      {
+        name: this.i18n('Current Size'),
+        prop: 'bucket_size',
+        flexGrow: 1,
+        pipe: this.dimlessBinaryPipe
+      },
+      {
+        name: this.i18n('No. of Objects'),
```

Comment:
Just seen how about `Objects`? (Than no one get's confused about the other template variables thinking of numbers ;) and not bytes)
```suggestion
        name: this.i18n('Objects'),
```